buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -332,6 +332,12 @@ class BeamModulePlugin implements Plugin<Project> {
       "--environment_cache_millis=10000",
```

Comment:
I assume the Go SDK doesn't have this flag to pass through the pipeline options yet. We may want to consider it in the future.

Reply:
Correct, and yeah it would probably be a good idea.